packages/sdk/src/utils/helpers.ts

```diff
@@ -0,0 +1,3 @@
+export function isFrontEnd() {
```

Comment:
We should comment this function and explain why its here. We should also say that this is not a safe function that is reliable.